cdap-gateway/src/main/java/co/cask/cdap/gateway/router/RouterMain.java

```diff
@@ -117,6 +118,7 @@ static Injector createGuiceInjector(CConfiguration cConf) {
       new DiscoveryRuntimeModule().getDistributedModules(),
       new RouterModules().getDistributedModules(),
       new SecurityModules().getDistributedModules(),
+      new SecureStoreModules().getDistributedModules(),
```

Comment:
Instead, just add your path in RouterPathLookup, just like how the authorization path is added